network-api/networkapi/wagtailpages/templates/wagtailpages/campaign_page.html

```diff
@@ -58,16 +58,18 @@ <h1 class="tw-h1-heading {% if uses_menu %}tw-hidden medium:tw-block{% endif %}"
 
                     {% block cta %}
                         {% if page.cta != None %}
-                            <div class="cta tw-pb-4
+                            <div class="cta tw-pb-4 tw-mx-auto
                                         {% if page.specific.narrowed_page_content == False %}
                                             large:tw-w-4/12 large:tw-ml-[8.3%] tw-px-8
                                         {% else %}
                                             tw-w-full
                                         {% endif %}"
                             >
-                                <div class="narrow-sticky-button-container hidden">
-                                    <a href="#cta-anchor" class="tw-btn-primary tw-block">{% trans "TAKE ACTION" context "Sticky button in mobile view on campaign pages" %}</a>
-                                </div>
+                                {% block cta_sticky_button %}
```

Comment:
Design requested that these pages do not have the sticky "take action" button that renders at the bottom of the page on mobile/tablet screens.

Adding this new `{% block cta_sticky_button %}` allows us to override this block on `app_install_page.html`